exporter/signalfxexporter/exporter_test.go

```diff
@@ -1318,6 +1318,7 @@ func TestDefaultSystemCPUTimeExcludedAndTranslated(t *testing.T) {
 }
 
 func TestTLSAPIConnection(t *testing.T) {
+	t.Skip("Flaky test see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/29099")
```

Comment:
```suggestion
```
Don't need this anymore, done in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29103

Reply:
[That's](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29103) a PR to this branch since I cannot push directly to this branch.